CONTRIBUTING.md

```diff
@@ -150,3 +150,24 @@ VERSION=preminor \
 Prereleases will automatically be published to npm with the dist tag `next` and browser bundles are automatically uploaded to the CDN.
 
 The dist tag ensures that prereleases are not installed by unsuspecting users who do not specify a version – npm automatically adds the `latest` tag to a published module unless one is specified.
+
+### Release issues
+
+#### Failed to publish all packages
+
+Problem: `lerna publish` failed part-way through meaning not all packages were published.
+
+To remedy this:
+ - First work out whether a cdn upload is required as the check used in `release.sh` is no longer valid since the `lerna version` step took place.
```

Comment:
I think it's worth being super clear here and mention `FORCE_CDN_UPLOAD` relates to this step (it's kind of obvious but better safe than sorry!)

This feels kind of long but also clearer:

```suggestion
 - First work out whether a CDN upload is required by checking if `@bugsnag/js` has changes since the last release. This must be done manually because the check used in `release.sh` is no longer valid because the `lerna version` step took place. If a CDN upload is required, set the `FORCE_CDN_UPLOAD` environment variable to any value
```